apps/main/src/dex/components/PageRouterSwap/Page.tsx

```diff
@@ -118,7 +118,7 @@ export const PageRouterSwap = () => {
     <StyledQuickSwapWrapper variant="primary" shadowed>
       <BoxHeader className="title-text">
         <IconButton testId="hidden" hidden />
-        {t`Swap`}
+        <span style={{ color: 'white' }}>{t`Swap`}</span>
```

Comment:
Can we put the color into the parent? otherwise...
```suggestion
        <Box sx={{ color: 'var(--white)' }}>{t`Swap`}</Box>
```